nx_arangodb/classes/dict/node.py

```diff
@@ -322,7 +325,7 @@ def __getitem__(self, key: str) -> NodeAttrDict:
             raise KeyError(key)
 
         if vertex_db := vertex_get(self.graph, node_id):
-            node_attr_dict = self._create_node_attr_dict(vertex_db)
+            node_attr_dict = self._create_node_attr_dict(vertex_db["_id"], vertex_db)
```

Comment:
Minor, but for better readability: Above L280 we're using `node` as a variable name. We should do this also here then, e.g. in this case `node_db` instead of `vertex_db`.

Reply:
good catch, fixed in 8ab094b